### DISCUSSION TITLE
What is the best Sampler and Scheduler for Outpaint/Inpaint with SDXL?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [MoonRide303/Fooocus-MRE#131] [Q&A] [answered]
![Screenshot_21](https://github.com/MoonRide303/Fooocus-MRE/assets/13344308/4857afa3-9793-4728-894f-a957eb82dd4d) ![Screenshot_22](https://github.com/MoonRide303/Fooocus-MRE/assets/13344308/c0100885-1e52-4e70-aa86-4ab1a97c4767)

**Comment by aimarxjg** [comment, marked as answer]
3M SDE GPU+ KARRAS（SLOW！！！）
The speed is very slow on non 40 series graphics cards, I don't know why (especially KRS）, usually SIMPLE
The SDXL rendering mode performs well, while also taking into account the fastest speeds of RES and UPC (in webui or comfy)
